src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmEntity.java

```diff
@@ -7,37 +7,57 @@
 package org.gridsuite.merge.orchestrator.server.repositories;
 
 import lombok.Getter;
+import lombok.Setter;
 import lombok.ToString;
-import org.springframework.data.cassandra.core.mapping.PrimaryKey;
-import org.springframework.data.cassandra.core.mapping.Table;
 
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.Index;
+import javax.persistence.Table;
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.UUID;
 
 /**
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com
  */
+@Entity
 @Getter
+@Setter
 @ToString
-@Table("merge_igm")
+@Table(name = "merge_igm")
 public class IgmEntity {
 
-    @PrimaryKey
+    @EmbeddedId
     private IgmEntityKey key;
 
+    @Column(name = "status")
     private String status;
 
+    @Column(name = "networkUuid")
     private UUID networkUuid;
 
+    @Column(name = "caseUuid")
     private UUID caseUuid;
 
+    @Column(name = "replacingDate")
     private LocalDateTime replacingDate;
 
+    @Column(name = "replacingBusinessProcess")
     private String replacingBusinessProcess;
 
+    @Column(name = "boundary")
+    @ElementCollection
+    @CollectionTable(foreignKey = @ForeignKey(name = "igmEntity_boundaries_fk", foreignKeyDefinition = "FOREIGN KEY (IgmEntity_date, IgmEntity_processUuid, IgmEntity_tso) REFERENCES merge_igm ON DELETE CASCADE"), indexes = {@Index(name = "igmEntity_boundaries_idx", columnList = "IgmEntity_processUuid, IgmEntity_date, IgmEntity_tso")})
     private List<UUID> boundaries;
```

Comment:
Two attributes EQ_UUID et TP_UUID is maybe more optimized (size and acces database)

Reply:
Done in PR for migration to PostGres : https://github.com/gridsuite/merge-orchestrator-server/pull/50